octobot_commons/profiles/profile_data.py

```diff
@@ -35,15 +35,18 @@ class ProfileDetailsData(octobot_commons.dataclasses.FlexibleDataclass):
 class CryptoCurrencyData(octobot_commons.dataclasses.FlexibleDataclass):
     trading_pairs: list[str]
     name: typing.Union[str, None] = None
+    leverage: typing.Union[float, None] = None
     enabled: bool = True
 
 
 @dataclasses.dataclass
 class ExchangeData(octobot_commons.dataclasses.FlexibleDataclass):
     exchange_credential_id: typing.Union[str, None] = None
     internal_name: typing.Union[str, None] = None
+    exchange_type: str = constants.DEFAULT_EXCHANGE_TYPE
     exchange_id: typing.Union[str, None] = None
     proxy_id: typing.Union[str, None] = None
+    default_leverage: typing.Union[float, None] = None
```

Comment:
this is to set a default leverage for a bot

Reply:
Should we create a FutureExchangeData and put default leverage in it? As, if I remember well, margin trading also has a "leverage" value

Reply:
Up !